cineast-core/build.gradle

```diff
@@ -90,18 +88,15 @@ dependencies {
     api group: 'com.google.code.gson', name: 'gson', version: version_gson
 
     /** Some basic stuff (collections etc). */
-    api group: "net.sf.trove4j", name: "trove4j", version: version_trove4j
     api group: "com.google.guava", name: "guava", version: version_guava
     api group: "com.googlecode.javaewah", name: "JavaEWAH", version: version_javaewah
+    api group: 'com.carrotsearch', name: 'hppc', version: '0.9.1'
 
     /** Mockito for unit testing */
-    implementation 'org.mockito:mockito-inline:4.5.1'
+    implementation 'org.mockito:mockito-inline:4.6.1'
 
     /** Tensorflow (Java). */
-    if (DefaultNativePlatform.currentOperatingSystem.isMacOsX())
-        api group: "org.tensorflow", name: "tensorflow-core-platform", version: version_tensorflow
-    else
-        api group: "org.tensorflow", name: "tensorflow-core-platform-gpu", version: version_tensorflow
+    api group: "org.tensorflow", name: "tensorflow-core-platform-gpu", version: version_tensorflow
```

Comment:
Is there a reason we're reverting to not having macOS specific bindings? https://github.com/vitrivr/cineast/pull/325

Reply:
No, that was not intentional. Had that change already been merged? Might be a git merge issue on my end, I will have a look.